#!/usr/bin/env/python3

from exception import safe_input



print('this is a test for exceptions on input')
response = safe_input()
print('you just entered {}'.format(response))